driver/main.F90

```diff
@@ -41,6 +42,12 @@ program rdycore_f90
       PetscCallA(RDyGetNumLocalCells(rdy_, n, ierr))
       allocate(h(n), vx(n), vy(n), rain(n))
 
+      ! get information about boundary conditions
+      PetscCallA(RDyGetNumBoundaryConditions(rdy_, nbconds, ierr))
+      do ibcond = 0, nbconds-1
```

Comment:
@jeff-cohere While BC IDs (`ibcond`) in the C code start with a `0`, should the Fortran use 0- or 1-based indexing? I think we should use 1-based in Fortran.

Reply:
I agree that 1-based indexing in Fortran is best, since it's less confusing.